string umrf_graph_name
bool name_match_required
string[] targets
string umrf_graph_json

temoto_action_engine_ros2/UmrfGraphDiff[] umrf_graph_diffs
